### DISCUSSION TITLE
Relationship filter query

### DISCUSSION THREAD

**Discussion by thewebguy27** [pocketbase/pocketbase#3322] [Q&A]
Hi!

I am trying to get records based on filter from the expanded relationship is it possible this way ? I want to fetch carrier that are **is_billable** but could get the desired result and it returns empty.
![image](https://github.com/pocketbase/pocketbase/assets/62599763/2d75a4bb-b2e5-4c5a-b010-b13010c35d59)
![image](https://github.com/pocketbase/pocketbase/assets/62599763/b461f89f-b44b-4d41-a703-59ad07337f75)
![image](https://github.com/pocketbase/pocketbase/assets/62599763/084591fb-fea3-49a1-9ca0-3e8bcc7eaf84)

**Comment by ganigeorgiev** [comment]
I'm having difficulties understanding the code screenshots (_you can paste code directly by wrapping them with tripple backticks_) but note that the `filter` expression will filter **only** the main collection records (_it seems to be "numbers" in your case_), aka. `carrier.is_billable=true` will return only the "numbers" records that have "carrier" relation matching the condition.
